src/devices/Arduino/ArduinoAnalogController.cs

```diff
@@ -19,8 +19,7 @@ internal class ArduinoAnalogController : AnalogController
         private readonly IReadOnlyList<SupportedPinConfiguration> _supportedPinConfigurations;
 
         public ArduinoAnalogController(ArduinoBoard board,
-            IReadOnlyList<SupportedPinConfiguration> supportedPinConfigurations, PinNumberingScheme scheme)
-            : base(scheme)
+            IReadOnlyList<SupportedPinConfiguration> supportedPinConfigurations)
```

Comment:
Chatted during triage with @pgrawehr about this. Let's use next triage meeting to confirm that our intention is to break compat as opposed to introduce new constructors without the numbering scheme and obsoleting the old ones.

Reply:
Yes, it's binary breaking, but most of the adjusted ctors looked like

```csharp
public Hcsr501(int outPin, PinNumberingScheme pinNumberingScheme = PinNumberingScheme.Logical, GpioController? gpioController = null, bool shouldDispose = true)
        {
            _outPin = outPin;

            _shouldDispose = shouldDispose || gpioController is null;
            _controller = gpioController ?? new GpioController(pinNumberingScheme);
           // ....
       }
```

The call `new Hcsr501(1, PinNumberingScheme.Board);` can easily be replaced by `new Hcsr501(1, new GpioController(PinNumberingScheme.Board))` if one really wants to keep using the Board numbering scheme for now.

Reply:
https://github.com/dotnet/iot/issues/2326#issuecomment-2391781486